sodacitylabs/boring-framework
# DONT MERGE

---
name: Pull Request
about: Ask for code to be merged into master
---

**Describe your Code**
Refactors the CLI to use both the Interpreter and Command patterns.

**Expected behavior**
No regressions. The entire "Getting Started" guide should still function as is.

**How to test**
Steps to test and verify the code works as expected:
1. Run the automated tests in the PR
2. Go through the Getting Started guide and ensure it still works

# WIP

- [ ] - Jest tests for the new classes
- [ ] - auto register expressions?
- [ ] - use async function definitions
